src/client/ducks/fetch/index.js

```diff
@@ -101,3 +101,17 @@ export const fetchActionCreator =
       variables,
       callback,
     })
+
+export const singleFetchActionCreator =
```

Comment:
Slightly odd to change just one parameter type, we should think about better approach in the future ...